RadixWallet/Clients/DeepLinkHandlerClient/DeepLinkHandlerClient+Live.swift

```diff
@@ -0,0 +1,39 @@
+extension DeepLinkHandlerClient {
+	public static var liveValue: DeepLinkHandlerClient {
```

Comment:
if you enable Isolation check this will fail I think?

should be: 

```swift
public static let liveValue: Self = {
    Self(...)
}()
```

right?